routes/recordOverview.js

```diff
@@ -91,6 +91,26 @@ var route = function (req, res, next) {
 
         });
 
+        // Define default overview action modal settings in a format that jade can access easily.
+        req.linz.model.linz.formtools.overview.footerActions.forEach((action) => {
```

Comment:
In fact, this is duplicate code. Can you create a function to do this and run both the `footerActions.forEach` and `actions.forEach` against it?